libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -734,6 +734,13 @@ const AP_Param::GroupInfo NavEKF3::var_info2[] = {
     // @Units: m
     AP_GROUPINFO("GPS_VACC_MAX", 10, NavEKF3, _gpsVAccThreshold, 0.0f),
 
+    // @Param: OPTIONS
+    // @DisplayName: Optional EKF behaviour
```

Comment:
Normally the Display name should start with the feature or subsystem so a better name would be, "EKF3 Optional Behaviour".

For the Description we should remove the first part, "This controls optional EKF behaviour.".  The "This controls" part in particular is not necessary because all parameters control something.

Reply:
I've created a commit at the end of this branch to resolve this and fix two other formatting issues. https://github.com/rmackay9/rmackay9-ardupilot/commits/tridge-pr-ekf-gps-check-fix/